lib/index.ts

```diff
@@ -225,3 +250,12 @@ export function multicastStream(
 		);
 	});
 }
+
+/**
+ * Escape characters that have a special meaning in regular expressions.
+ * Borrowed from:
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+ */
+function escapeRE(s: string) {
```

Comment:
I guess `lodash` would be useful then, as there's a `_.escapeRegExp()` method that does exactly this.

Reply:
Well, after @zvin's comment, it turns out I don't need a regular expression at all...